### PR TITLE
Bring About page updates over

### PR DESCRIPTION
**Pull request by DrJKL**

### CODE REVIEW

File: content/about/index.md [hunk 1]
```diff
@@ -9,36 +9,44 @@ weight = 10
...
+### SVNA's Board
 
 |                        |               |
 | ---------------------- | ------------- |
 | **President**          | Bee Hanson    |
 | **Vice President**     | Leticia Jones |
 | **Treasurer**          | Randy Wise    |
 | **Communications**     | John Waters   |
-| **Guy Who Does Stuff** | Alex Brown    |
+| **Member At Large**    | Alex Brown    |
```
Comment (DrJKL) [inline content/about/index.md:44]: I still prefer my formal title

File: content/about/index.md [hunk 2]
```diff
@@ -9,36 +9,44 @@ weight = 10
...
 
 Email: <board@santiagovilla.org>
 
+Our current President is Bee Hanson, one of the original founders of the SVRA, and steadfast recorder of all SVNA and Mountain View Mobile Home Association (MVMHA) activities. Bee has lived in the neighborhood since 1996, and is looking forward to many more years here.
```
Comment (DrJKL) [inline content/about/index.md:48]: Not Chronicler?

File: content/about/index.md (hunk 1 above)
Comment (beelia) [reply content/about/index.md:44]: I shoulda waited until I could merge without you knowing

File: content/about/index.md (hunk 2 above)
Comment (beelia) [reply content/about/index.md:48]: Too cute

File: content/about/index.md (hunk 1 above)
Comment (DrJKL) [reply content/about/index.md:44]: And this is why we have branch protection. 😛

**Comment by beelia** [comment]
Rats. You win. I surrender. Member at Large it is.

Otherwise you'll be turning me into a Chronicler.

🐝

**Comment by DrJKL** [comment]
When we voted, the official title was "Guy Who Does Stuff Sometimes I Guess"

**Comment by DrJKL** [comment]
You're still the 𝕮𝖍𝖗𝖔𝖓𝖎𝖈𝖑𝖊𝖗 to me <3

**Comment by beelia** [comment]
😸

**Pull request merged**